MDANSE/Src/MDANSE/Framework/Jobs/DynamicIncoherentStructureFactor.py

```diff
@@ -274,6 +274,8 @@ def finalize(self):
 
         nAtomsPerElement = self.configuration["atom_selection"].get_natoms()
         weights = self.configuration["weights"].get_weights()
+        for key, value in weights.items():
+            weights[key] = value * value
```

Comment:
Equivalent but clearer?
```suggestion
            weights[key] = value ** 2
```